lib/index.js

```diff
@@ -1,4 +1,30 @@
 /*
  * Copyright (c) 2018 Veres One Project. All rights reserved.
  */
-module.exports = require('./did-veres-one');
+'use strict';
+
+const tls = require('tls');
+tls.DEFAULT_ECDH_CURVE = 'auto';
```

Comment:
Is this still needed? Is it appropriate for this lib? I think this was a hold over from the cli.

Reply:
Good point, I can probably remove it.

Reply:
Removed.